controller/handler/dashboard.js

```diff
@@ -15,41 +15,83 @@ const database = require('../../model');
  * @returns {View} Rendered page
  */
 function dashboardView (request, reply) {
-    const trial = database.sequelize.model('trial');
+    const todayDate = moment();
 
-    trial
-    .findAll()
-    .then((trials) => {
-        // Process data into format expected in view
-        const trialData = trials.map(processTrial);
+    return Promise
+        .all([
+            database.sequelize.query(
```

Comment:
since there is only 1 query, return `database.sequelize.query` without the `Promise.all` wrapper